cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/DefaultTicketRegistry.java

```diff
@@ -47,7 +47,7 @@
  * @since 3.0.0
  */
 @Component("defaultTicketRegistry")
-public final class DefaultTicketRegistry extends AbstractTicketRegistry implements Job {
+public final class DefaultTicketRegistry extends AbstractCrypticTicketRegistry implements Job {
```

Comment:
As the `DefaultTicketRegistry` lies in memory, what's the advantage of encrypting tickets? As it's mostly used in development, it makes debugging harder.